packages/meteor-gazelle-ipmanager/lib/loginAttempts.js

```diff
@@ -0,0 +1,56 @@
+LoginAttempts = new Mongo.Collection('loginAttempts');
+LoginAttempt = Astro.Class({
+  name: 'LoginAttempt',
+  collection: LoginAttempts,
+  fields: {
+    ip: {
+      type: 'number',
+      index: 1
```

Comment:
You can, in regards to standardizing I would say we explicitly say which fields are indexes (as seen the BannedIp collection). I would argue for this because composite indexes will need to be placed in that specific sub-object.